akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -729,6 +730,11 @@ private[remote] abstract class ArteryTransport(_system: ExtendedActorSystem, _pr
       associationRegistry.setUID(peer).completeHandshake(peer)
     } catch {
       case ShuttingDown => Future.successful(Done) // silence it
+      case exc: UidCollisionException =>
+        log.error(exc, exc.getMessage)
```

Comment:
Won't this just log the message twice, is the stack trace very interesting, now that we make it non critical, could it just be a warning message saying something like "UID collision peer {} uid {}"?

Reply:
you mean that it's twice because of the logging from `quarantine`. That is true, but the `quarantine` logging will only be on INFO because the uid is unknown (handshake not completed). I can change to warning and skip the stack trace. I think it can be good to be somewhat noisy since I think the other side will not really notice and will retry until there is a handshake timeout or such. Collisions are not supposed to happen often (we have never seen it reported before).

Reply:
Ok, maybe leave as error, but skip the stacktrace then?